README.md

```diff
@@ -100,7 +107,9 @@ Resources:
 
 # Releasing
 ## Windows & Linux
-We use [CircleCI](https://circleci.com/gh/nos/client) to automatically create builds based upon git tags.
+We use 
```

Comment:
The line breaks throughout here are kind of random.  It doesn't read like a sentence anymore.  Could we just include the color change itself?

Reply:
You're mistaken.

Reply:
`The markup will ignore the line break in the plaintext, where it helps singling out and working on the link. This holds for many other text formatters as well btw.`

I'm not following, where does this provide any difference for github *or* text formatters in comparison to keeping it all on the same line?

Reply:
On the user side, it will look the same. On the editor side, the first looks like

---

"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book, see **| lipsum reference link name |(https://www.lipsum.com/).** It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum."

---

and the second will look like

---

"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book, see 
**| lipsum reference link name |(https://www.lipsum.com/)**.
It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum."

Reply:
I don't see a problem with the first, honestly.

Reply:
I'd argue it's more work to edit if you have to search for the links, e.g. when some inevitably break. But it's just a style choice suggestion and so let's stick to the unedited way if it works for you.